lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -888,12 +888,18 @@ class SqlParser(
                     // TODO:  we are losing case-sensitivity of the function name here.  Do we care?
                     call(idArg.name.text, args.drop(1), metas)
                 }
+                "union" -> union(distinct(), args, metas)
```

Comment:
We also need some tests (probably in [SqlParser.kt](https://github.com/partiql/partiql-lang-kotlin/blob/0b271115b44b74b63aa1e7207d6f23d2da89b630/lang/test/org/partiql/lang/syntax/SqlParserTest.kt)) to make sure the `PartiqlAst` output by the parser is correct.